### ISSUE TITLE
MA0011: false positives reported

### ISSUE THREAD

**Issue by MaceWindu** [meziantou/Meziantou.Analyzer#684]
### Version of the Meziantou.Analyzer NuGet package

2.0.136

### Rule Identifier

MA0011

### Target Framework

any

### C# Language version

C#12

### Description

Because `IFormatProvider` not used by methods below, they shouldn't be reported

### Reproduction Steps

Minimal code:

```c#
Convert.ToChar("0");
Convert.ToBoolean("true");
```
